app/assets/javascripts/application.js

```diff
@@ -29,3 +29,8 @@
 //= require underscore_mixins
 //= require jstimezonedetect
 //= require_tree .
+
+// initialize popovers after all of our other javascript is done
+$(function () {
+  $('[data-toggle="popover"]').popover()
+})
```

Comment:
maybe not start the junk-drawer ... separate file ?

Reply:
the file will have to have a dumb name, because `require_tree .` needs to be the last directive so we need to make sure its the last thing in the tree.